sr_description/hand/xacro/full_hand.urdf.xacro

```diff
@@ -1,81 +1,76 @@
   <xacro:include filename="$(find sr_description)/hand/xacro/forearm/forearm.urdf.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/wrist/wrist.urdf.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/palm/palm.urdf.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/finger/fingers.urdf.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/thumb/thumb.urdf.xacro" />
-  <xacro:property name="pi" value="3.1415926535897931" />
-  <!-- muscletrans is true for muscle transmission to be used, false to use motor transmission -->
-  <!-- muscle is true for muscle hand or false for motor hand -->
-  <!-- bio is true for biotac finger or false for standard finger -->
-  <!-- bt_sp is true for BT_SP finger or false for standard finger -->
-  <!-- ubi is true for ubi finger or false for standard finger -->
-  <!-- eli is true for ellipsoid fingertip or false for standard finger -->
+  <!-- hand_type can be: hand_e, hand_e_plus, hand_m (muscle), hand_mt (muscletrans) -->
+  <!-- fingers: dict specifying available fingers, default: dict(th=1, ff=1, mf=1, rf=1, lf=1) -->
+  <!-- tip_sensors: dict specifying the fingertip (tactile) sensors per finger: std, none, bio, bt_sp, ubi, eli -->
+  <!-- mid_sensors: dict specifying the middle-phalange (tactile) sensors per finger: std, none, bio, ubi -->
+  <!-- prox_sensors: dict specifying the proximal (tactile) sensors per finger: std, none, ubi -->
+  <!-- palm_sensor: specifies the palm tactile sensor: std, none, ubi -->
   <!-- reflect is either 1 (for right hand) or -1 (for left hand) -->
   <!-- prefix can be anything but usually is "r_" or  "l_" for right and left hands distinction -->
-  <!-- lf is true if we want a little finger, false otherwise -->
-  <xacro:macro name="shadowhand"
-  params="muscletrans muscle hand_type:='hand_e' bio bt_sp ubi eli reflect prefix lf cable_mesh:='false'">
-    <!--xacro:property name="hand_type" value="hand_e"/-->
+  <!-- muscletrans (deprecated) is true for muscle transmission to be used, false to use motor transmission -->
+  <!-- muscle (deprecated) is true for muscle hand or false for motor hand -->
+  <!-- bio (deprecated) is true for biotac finger or false for standard finger -->
+  <!-- bt_sp (deprecated) is true for BT_SP finger or false for standard finger -->
+  <!-- ubi (deprecated) is true for ubi finger or false for standard finger -->
+  <!-- eli (deprecated) is true for ellipsoid fingertip or false for standard finger -->
+  <!-- lf (deprecated) is true if we want a little finger, false otherwise -->
+  <xacro:macro name="shadowhand" params="hand_type:='hand_e' reflect prefix cable_mesh:=False fingers:=${[1,1,1,1,1]}
+          tip_sensors:=${5 * ['std']} mid_sensors:=${5 * ['std']} prox_sensors:=${5 * ['std']} palm_sensor:=std
```

Comment:
```suggestion
          tip_sensors:=${5 * ['std']} mid_sensors:=${5 * ['std']} prox_sensors:=${5 * ['std']} palm_sensor:='std'
```

Reply:
Can I then ask why `hand_type:='hand_e'` needs the quotes ? I find that with the quotes it is clearer

Reply:
None of these needs quotes. Thanks for pointing these out. I will push a cleanup commit to remove those everywhere.